README.md

```diff
@@ -68,82 +69,161 @@ python -m pip install git+https://github.com/huggingface/optimum.git#egg=optimum
 
 Check out the examples below to see how 🤗 Optimum can be used to train and run inference on various hardware accelerators.
 
-### Accelerated training
+## Accelerated inference
```

Comment:
```suggestion
### Accelerated inference
```
To be consistent witht he quick tour paragraph.

Reply:
This was modified on purpose in order to increase the size of the inference and training section, otherwise I feel it's not visible enough du to all the nested sections